ci/release/update-version.sh

```diff
@@ -56,3 +56,8 @@ for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
 done
 sed_runner "s/RAPIDS_VERSION_NUMBER=\".*/RAPIDS_VERSION_NUMBER=\"${NEXT_SHORT_TAG}\"/g" ci/build_docs.sh
+
+# Docs referencing source code
+for FILE in docs/source/*.rst; do
+    sed_runner "s/branch-${CURRENT_SHORT_TAG}\//branch-${NEXT_SHORT_TAG}\//g" "${FILE}"
```

Comment:
*question*: I believe the `main` branch always points at the current stable release branch so could one point at that? Or is this to handle nightly branches too?

Reply:
No, I think this is fine, since we want both styles of links.